libnvidia-container.yaml

```diff
@@ -0,0 +1,83 @@
+package:
+  name: libnvidia-container
+  version: 1.15.0
+  epoch: 0
+  description: NVIDIA container runtime library
+  copyright:
+    - license: Apache-2.0
+    - license: GPL-3.0-only
+    - license: LGPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - automake
+      - bmake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils # for GNU date
+      - curl
+      - glibc-dev
+      - go
+      - libcap-dev # for sys/capability.h
+      - libseccomp
+      - libseccomp-dev
+      - libtirpc-dev # for rpc/rpc.h
+      - libtool
+      - linux-headers
+      - lsb-release-minimal
+      - m4
+      - nfs-utils # for rpcgen
+      - patch
+      - pkgconf-dev
+  environment:
+    CPATH: /usr/include/tirpc
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/libnvidia-container
+      tag: v${{package.version}}
+      expected-commit: 6c8f1df7fd32cea3280cf2a2c6e931c9b3132465
+
+  # Fix prefix to /usr instead of /usr/local
+  - runs: sed -i 's|export prefix *= */usr/local|export prefix = /usr|' Makefile
```

Comment:
just checking, does `- uses: autoconf/configure` work here instead of using `sed`?

Reply:
Unfortunately, repo doesn't provide a `./configure` file in root dir.